Package.swift

```diff
@@ -29,12 +29,14 @@ let package = Package(
       name: "FirebaseAnalytics",
       targets: ["FirebaseAnalyticsTarget"]
     ),
-    // This library is not designed to be imported into client source code.
+    // Adding this library to the your project is enough for it to take effect. The module
```

Comment:
"Adding this library **to your** project is enough for it to take effect. The module"
or
"Adding this library **to the** project is enough for it to take effect. The module"